chrome-extension/content.js

```diff
@@ -120,23 +130,61 @@ function customCreateElement(type, classList, innerHTML = '') {
 const serverUrl = 'http://localhost:8080';
 // TODO: select between dev host and prod host
 
-async function addItem() {
+/**
+ * Returns a Promise that resolves to a <select> element,
+ * a multi-select dropdown menu of category options from fetching db
+ * @param {String} userId 
+ */
+async function getCategoryDropdown(userId) {
+  const categoryDropdown = customCreateElement('select', []);
+  categoryDropdown.setAttribute('multiple', true);
+
+  const defaultOption = customCreateElement('option', [], 'Option: Select a Category');
```

Comment:
The formatting I'm getting for this element is a bit odd:

![image](https://user-images.githubusercontent.com/11233666/88295480-79ac8c00-cccb-11ea-8021-f943a40726f8.png)

Can we get the add clipping button to be in the sidebar and show only the "select a category" option until the user selects this element?

Reply:
discussed